.travis.yml

```diff
@@ -1,15 +1,26 @@
+
 language: node_js
 node_js: "stable"
 
+python:  3.6
+cache: pip
+
 before_install:
   - npm install -g npm@latest
+  - pip install --upgrade pip
```

Comment:
this line is redundant with the next one

Reply:
That's after that I realized. *Laughs*